src/background.js

```diff
@@ -116,9 +121,22 @@ const rollout = {
     await browser.study.setup(baseStudySetup);
     browser.runtime.onMessage.addListener((...args) => this.handleMessage(...args));
   },
+  async showTab() {
+    const tabs = await this.findStudyTabs();
```

Comment:
What is this mechanism for, exactly?

Reply:
Just to prevent multiple of the same tabs and focus instead if one exists.